CONTRIBUTING.md

```diff
@@ -11,6 +11,12 @@ Endo is a yarn workspaces repository. Running yarn in the root will install and
 Note: running yarn `--ignore-scripts` will not complete the setup of SES.
 Note: Endo uses `lerna` only for releasing. `lerna bootstrap` is unlikely to work.
 
+## Creating a new package
+
+Run <code>[scripts/create-package.sh](./scripts/create-package.sh) $name</code>,
+then update the resulting README.md, package.json (specifically `description`),
+index.js, and index.test.js files.
```

Comment:
```suggestion
then update the resulting `README.md`, `package.json` (specifically `description`),
`index.js`, and `index.test.js` files.
```

Reply:
Formatting file names as code generally seems a bit off to me. We can revisit later, but for now I'll be leaving this as-is.